api/server/runner.go

```diff
@@ -208,7 +207,7 @@ func (s *Server) serve(ctx context.Context, c *gin.Context, appName string, foun
 		c.JSON(http.StatusAccepted, map[string]string{"call_id": task.ID})
 
 	default:
-		result, err := runner.RunTask(s.tasks, ctx, cfg)
+		result, err := Api.Runner.RunTask(ctx, cfg)
```

Comment:
Yes, I agree we need a solution for that `Api` global variable. But we are actually refactoring that right now?

I mean, we could also refactor that in this PR and solve all that problem. But I think that wouldn't change the objective of this PR that is simplify and make the server instantiation simpler, mainly for users.

This PR is just fixing part of the problem, but we could also refactor everything to solve the hole problem.